docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -65,19 +77,50 @@ You can now go ahead and click `Review and launch`.
 
 You will need to enable enhanced health checks for your Beanstalk Monitoring.
 
-Click on the `Modify` link under the Monitoring section as seen here:
+Click on the `Edit` link under the Load Balancer section as seen here:
+
+![Elastic Beanstalk Monitoring](images/EBLoadBalancerEdit.png)
+
+The `Health check path` is how Elastic Beanstalk knows when the application is ready to run. This path is also used by the Load Balancer to check application heath. Set this path to `/api/health` in the Processes section.
+
+![Elastic Beanstalk Monitoring Process](images/EBProcessesSection.png)
+
+Select the default process and click on `Actions` -> Edit and change the Path from `/` to `/api/health`
+
+![Elastic Beanstalk Monitoring endpoint](images/EBProcessEditEndpointHealth.png)
+
+After configuring this health check you can click on `Save` at the bottom of the page
+
+#### Enabling VPC
+
+You must enable your Application to exist in a VPC unless you will receive an error when creating it. To use a VPC, head to the Network section in the configuration and click on the `Edit` button:
```

Comment:
We might want to include a one-liner about what a VPC is (and a link to learn more), and why we need to enable it (beyond just "you'll receive an error").

Reply:
done! thank!